rollup.config.ts

```diff
@@ -34,7 +41,7 @@ const buildEsm = {
     'src/graphql.ts',
     'src/context/index.ts',
   ],
-  external: ['debug'],
+  external: ['debug', mswjsRegex],
```

Comment:
What do you think if we name this variable for what it represents intention-wise rather than what data type it holds?

```diff
-mswjsRegex
+ecosystemDependencies
```

Reply:
if we're going to go that far, you should just externalize all of your node_modules (which is generally standard practice for libraries). This feels scary and out of scope for the small bug I'm personally trying to fix and get released.

Reply:
but sure, I can update the variable name...

Reply:
Future work idea: use https://www.npmjs.com/package/rollup-plugin-auto-external to automatically exclude all "dependencies" from the esm/node bundles.